doc/HOWTO_RELEASE.rst

```diff
@@ -37,33 +29,29 @@ are supported; For the first half of 2020, this will be Python >= 3.6. We test
 NumPy against all these versions every time we merge code to main.  Binary
 installers may be available for a subset of these versions (see below).
 
-OS X
-----
-OS X versions >= 10.9 are supported, for Python version support see
-:ref:`NEP 29 <NEP29>`. We build binary wheels for
-OSX that are compatible with Python.org Python, system Python, homebrew and
-macports - see this `OSX wheel building summary
-<https://github.com/MacPython/wiki/wiki/Spinning-wheels>`_ for details.
+* **OS X**
 
+  OS X versions >= 10.9 are supported, for Python version support see
+  :ref:`NEP 29 <NEP29>`. We build binary wheels for OSX that are compatible with
+  Python.org Python, system Python, homebrew and macports - see this
+  `OSX wheel building summary <https://github.com/MacPython/wiki/wiki/Spinning-wheels>`_
+  for details.
 
-Windows
--------
-We build 32- and 64-bit wheels on Windows. Windows 7, 8 and 10 are supported.
-We build NumPy using the `mingw-w64 toolchain`_ on Appveyor.
+* **Windows**
 
+  We build 32- and 64-bit wheels on Windows. Windows 7, 8 and 10 are supported.
+  We build NumPy using the `mingw-w64 toolchain`_ on Appveyor.
```

Comment:
"on Appveyor" -> "using cibuildwheels and GitHub Actions."

We could maybe link to the cibuildwheels documentation: https://cibuildwheel.readthedocs.io/en/stable/